AUDIO_GET_PTS and VIDEO_GET_FRAME_RATE are removed with linux-libc-headers 4.19.x
Define AUDIO_GET_PTS and VIDEO_GET_FRAME_RATE, based on:
https://elixir.bootlin.com/linux/v3.2/source/include/linux/dvb/video.h#L255
and
https://elixir.bootlin.com/linux/v4.9.12/source/include/uapi/linux/dvb/audio.h#L132

diff --git a/lib/dvb/decoder.cpp b/lib/dvb/decoder.cpp
index 3459a9cc6..9f77d42bf 100644
--- a/lib/dvb/decoder.cpp
+++ b/lib/dvb/decoder.cpp
@@ -24,6 +24,9 @@
 
 DEFINE_REF(eDVBAudio);
 
+#define AUDIO_GET_PTS			_IOR('o', 19, __u64)
+#define VIDEO_GET_FRAME_RATE	_IOR('o', 56, unsigned int)
+
 eDVBAudio::eDVBAudio(eDVBDemux *demux, int dev)
 	:m_demux(demux), m_dev(dev)
 {
